npm-debug.log

```diff
@@ -0,0 +1,112 @@
+0 info it worked if it ends with ok
```

Comment:
This file shouldn't be uploaded. Please add the file name in .gitignore.

Reply:
done.